### PR TITLE
Update quickstart.py

### PR DESCRIPTION
**Pull request by cosmicwanderer7**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Do not include any personal data.

Fixes # (issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Test

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project, `black -t py34`
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by Octoangel — COMMENTED** [review]
Hiii

**Comment by Octoangel** [comment]
do you use instagram

**Comment by cosmicwanderer7** [comment]
> do you use instagram

yes

**Review by cosmicwanderer7 — COMMENTED** [review]
have changed few things in quick start

**Review by prithvijeet7000 — APPROVED** [review]

**Comment by cosmicwanderer7** [comment]
> do you use instagram

can you review my changes

**Comment by CodeMasterBond** [comment]
Even if @prithvijeet7000 reviews the changes, your pull request will still not be merged as it requires at least 1 approving review by reviewers with write access.

File: quickstart.py [hunk 1]
```diff
@@ -10,7 +10,13 @@
 set_workspace(path=None)
 
 # get an InstaPy session!
-session = InstaPy()
+your_username = ' ' #input your username here
```
Comment (elulcao) [inline quickstart.py:13]: Please check black format:

```bash
# get an InstaPy session!
your_username = " "  # input your username here
your_password = " "  # input your password here

session = InstaPy(
    username=your_username, password=your_password, headless_browser=False
)  # headless browser if set True will run without displaying mozilla firefox in background
```

Comment (cosmicwanderer7) [reply quickstart.py:13]: Okay

**Pull request closed without merging**